daily_reports.rb

```diff
@@ -36,6 +36,7 @@ def all_projects(date, slack, text, rerun, verbose, customer_facing, short)
       error = <<~MSG
       Generation of daily report for project *#{project.name}* stopped due to error:
       #{e}
+      #{e.error_messages}
```

Comment:
Just noticed that for this (but not when run for an individual project), the error messages array is being printed. E.g. 
```
Timeout error querying daily cost Azure API for project azure. All 3 attempts timed out.
["Attempt 1:\nError code 504.\n", "Attempt 2:\nError code 504.\n", "Attempt 3:\nError code 504.\n"]
```
Also means if no error messages (say for an AWS project) an empty array is printed.

Reply:
It seems that IRB has lied to me.